src/App.js

```diff
@@ -9,50 +9,60 @@
     WARNING: You need to transform the facts variable into a state.
 */
 
+import { useEffect, useState } from "react";
 import { Container, ListGroup, ListGroupItem, Navbar } from "react-bootstrap";
 
 async function getNewFact() {
-    const response = await fetch("https://catfact.ninja/fact");
-    const obj = await response.json();
+  const response = await fetch("https://catfact.ninja/fact");
+  const obj = await response.json();
 
-    return obj.fact;
+  return obj.fact;
 }
 
 export default function App() {
-    // Use state here
-    // Hint: https://reactjs.org/docs/hooks-state.html
-    // Hint: https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly
-    const facts = [
-        "Cats are weird",
-    ];
-
-    // Using the getNewFact function, load a new fact when this component is first loaded
-    // Hint: https://reactjs.org/docs/hooks-effect.html
-
-    return (
-        <Container>
-            <Navbar as="header" bg="primary" className="bg-gradient">
-                <Container fluid>
-                    <Navbar.Brand className="text-white">Tema React 2</Navbar.Brand>
-                </Container>
-            </Navbar>
-
-            <Container as="main" className="my-5 flex-grow-1">
-                <ListGroup>
-                    <ListGroup.Item className="fw-bold">Cat facts</ListGroup.Item>
-                    {facts.map((fact) => <ListGroupItem key={fact}>{ fact }</ListGroupItem>)}
-                    
-                    <ListGroup.Item action={true} variant="primary">
-                        {/* 
+  // Use state here
+  // Hint: https://reactjs.org/docs/hooks-state.html
+  // Hint: https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly
+  const [fact, setFact] = useState("");
+  useEffect(() => {
+    (async () => {
+      const response = await fetch("https://catfact.ninja/fact");
+      const obj = await response.json();
+      setFact(obj.fact);
```

Comment:
Ai exact acești 3 pași în funcția `getNewFact`

```suggestion
      setFact(await getNewFact());
```